CONTRIBUTING.md

```diff
@@ -21,6 +23,38 @@ sending a pull request. Please follow these guidelines:
     document the setting. The comments in this file are extracted and turned into documentation. Pay
     attention to the comment formatting and follow the existing style closely.
 
+## Development Environment
+
+### Prerequisites 
+
+On Windows, make sure to set your PowerShell execution policy as shown [here](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.security/set-executionpolicy?view=powershell-5.1),
+otherwise the PowerShell tests will fail.
+
+### Setting Up
+
+To begin development on Rez you'll first need to set up your development environment. There are many 
+ways you can do it, but this is the recommended approach. This approach will automatically create a virtual
+environment for you, patch the Rez binaries, and copy completion scripts. All tests will be run this way.
+
+1. Fork the repo and clone it.
+2. Create a new Git branch and check it out.
+3. Install your local Rez code by running `python install.py venv`.
+4. Activate the virtual environment by running the `activate` file.
+
+There is an alternative method of setting up your development environment, that doesn't use the `install.py`
+script. This method is not recommended however, as not all tests will be run this way.
+
+1. Fork the repo and clone it.
+2. Create a new Git branch and check it out.
+3. Create a virtual environment in the same directory as the repo by running `python -m venv venv`.
+4. Activate the virtual environment by running the `activate` file.
+5. Pip install your local Rez code by running `pip install .`
+
+### Running Tests
+
+1. Set up your development environment as shown above.
+2. Run `rez selftest`.
```

Comment:
When installing with the install script, this won't work. The rez command won't be in `$PATH`.

Reply:
@JeanChristopheMorinPerso Yeah I see when you pip install Rez the binaries get placed directly in the "Scripts" folder on Windows, whereas when using Rez's `install.py` file, Rez binaries are placed in the "Scripts/rez" folder, so they are not automatically found by just activating the virtual environment :/

I guess I can just add another step to the `install.py` method that mentions you need to manually add the "Scripts/rez" folder to $PATH?

Reply:
Yep! I would also mention that we recommend running the tests with `pytest` and the `parameterized` package.